content/plugins/environment-plugin.md

```diff
@@ -6,63 +6,74 @@ contributors:
 ---
 
 The EnvironmentPlugin is a shorthand for using the [DefinePlugin](/plugins/define-plugin) on [`process.env`](https://nodejs.org/api/process.html#process_process_env) keys.
-```js
-new webpack.DefinePlugin({
-  'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
-})
-```
-can be replaced with
-```js
-new webpack.EnvironmentPlugin([ 'NODE_ENV' ])
-```
 
+## Usage
 
-## How to use
-The EnvironmentPlugin accepts either an array of keys or an object with default values.
+The EnvironmentPlugin accepts either an array of keys.
 
-```js
-new webpack.EnvironmentPlugin(['NODE_ENV', 'VERSION'])
+```javascript
+new webpack.EnvironmentPlugin(['NODE_ENV', 'DEBUG'])
 ```
 
-When given an object, the values are used as defaults for when they are not defined in `process.env`.
+This is equivalent to the following DefinePlugin application:
```

Comment:
`DefinePlugin` over DefinePlugin. You could do the same change for the rest as it reads a little better then.